src/integration-test/integration/MappingsClient.test.ts

```diff
@@ -0,0 +1,485 @@
+    expect(calcProperty).to.not.be.undefined;
+
+    //create customCalculations
+
+    const newCustomCalculation: CustomCalculationCreate = {
+      propertyName: "cust1",
+      formula: "1+1",
+      quantityType: QuantityType.Distance,
+    };
+    let custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+    customCalculationId = custCalculation.id;
+
+    newCustomCalculation.propertyName = "cust2";
+    custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+
+    newCustomCalculation.propertyName = "cust3";
+    custCalculation = await mappingsClient.createCustomCalculation(accessToken, testIModel.id, mappingIds[0], groupId, newCustomCalculation);
+    expect(custCalculation).to.not.be.undefined;
+  });
+
+  after(async () => {
+    let response: Response;
+    while(mappingIds.length > 0) {
+      response = await mappingsClient.deleteMapping(accessToken, testIModel.id, mappingIds.pop() ?? "");
+      expect(response.status).to.be.eq(204);
+    }
+    await testIModelGroup.cleanupIModels();
+  });
+
+  //mapping tests
+  it("General - get all with iterator", async function () {
+    const iterator = mappingsClient.getMappingsIterator(accessToken, testIModel.id);
+    let mapping = await iterator.next();
+    do {
+      expect(mapping.value).to.not.be.undefined;
+      mapping = await iterator.next();
+    } while(!mapping.done);
+  });
+
+  it("General - fail request", async function () {
+    await expect(mappingsClient.getMappings(accessToken, "-")).to.be.rejected;
+  });
+
+  it("Mappings - Create and delete", async function () {
+    const newMapping: MappingCreate = {
+      mappingName: "Test",
+    }
+    const mapping = await mappingsClient.createMapping(accessToken, testIModel.id, newMapping);
+    expect(mapping).to.not.be.undefined;
+
+    const response = await mappingsClient.deleteMapping(accessToken, testIModel.id, mapping.id);
+    expect(response.status).to.be.eq(204);
+  });
+
+  it("Mappings - Update", async function () {
+    const mappingUpdate: MappingUpdate = {
+      description: "Updated description"
+    }
+    const mapping = await mappingsClient.updateMapping(accessToken, testIModel.id, mappingIds[0], mappingUpdate);
+    expect(mapping).to.not.be.undefined;
+  });
+
+  it("Mappings - Copy", async function () {
+    const mappingCopy: MappingCopy = {
+      targetIModelId: testIModel.id,
+      mappingName: "MappingCopy",
+    };
+    const mapping = await mappingsClient.copyMapping(accessToken, testIModel.id, mappingIds[0], mappingCopy);
+    expect(mapping).to.not.be.undefined;
+
+    const response = await mappingsClient.deleteMapping(accessToken, testIModel.id, mapping.id);
+    expect(response.status).to.be.eq(204);
```

Comment:
Add more assertions to all tests. At least check if the mapping name is set on the returned mapping.

Reply:
All entities now have at least one property that is checked